BraveWallet/Crypto/NetworkPicker.swift

```diff
@@ -14,17 +14,33 @@ extension BraveWallet.EthereumChain {
 }
 
 struct NetworkPicker: View {
-  var networks: [BraveWallet.EthereumChain]
+  @ObservedObject var networkStore: NetworkStore
   @Binding var selectedNetwork: BraveWallet.EthereumChain
+  @State private var isPresentingNetworkList: Bool = false
+  @Environment(\.presentationMode) @Binding private var presentationMode
+  @Environment(\.buySendSwapDestination) private var buySendSwapDestination
```

Comment:
Make this `@Binding private var buySendSwapDestination` like presentation mode

Reply:
but i'm only need to read the `buySendSwapDestination.wrappedValue` not change it

Reply:
It just means you don't have to read into `wrappedValue` (e.g. `if buySendSwapDestination == x`)